mappings/net/minecraft/client/render/entity/model/PlayerEntityModel.mapping

```diff
@@ -14,4 +14,12 @@ CLASS net/minecraft/class_591 net/minecraft/client/render/entity/model/PlayerEnt
 	METHOD method_22697 getRandomPart (Ljava/util/Random;)Lnet/minecraft/class_630;
 		ARG 1 random
 	METHOD method_2823 renderCape (Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;II)V
+		ARG 1 matrices
+		ARG 2 vertices
+		ARG 3 light
+		ARG 4 overlay
 	METHOD method_2824 renderEars (Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;II)V
+		ARG 1 matrices
+		ARG 2 vertices
```

Comment:
class_4587

Reply:
If it's a VertexConsumer naming it `vertices` is intended.